org/party.go

```diff
@@ -29,6 +29,8 @@ type Party struct {
 	Addresses []*Address `json:"addresses,omitempty" jsonschema:"title=Postal Addresses"`
 	// Electronic mail addresses
 	Emails []*Email `json:"emails,omitempty" jsonschema:"title=Email Addresses"`
+	//
+	Websites []*Website `json:"websites,omitempty" jsonschema:"title=Websites"`
```

Comment:
The name `Websites` felt the most appropriate here as it feels more explicit compared to something like "links".